piu/cli.py

```diff
@@ -143,7 +148,25 @@ def ssh_keys_added():
         return False
 
 
-def _request_access(even_url, cacert, username, hostname, reason, remote_host, lifetime, clip, connect, tunnel):
+def _request_eic_access(
+    ec2, hostname: str, remote_host: str, clip: bool, connect: bool, tunnel: bool, ssh_public_key: str
+) -> bool:
+    host_via = hostname
+    click.secho("Requesting access with EIC to host {host_via}".format(host_via=host_via), bold=True)
+    if remote_host:
+        try:
+            remote_attributes = instance_attributes(ec2, "private-ip-address", remote_host)
+        except RuntimeError:
+            print("Failed to get attributes for instances with private IP address {}".format(remote_host))
+            return False
+        if not send_ssh_key("ubuntu", remote_attributes, ssh_public_key):
+            return False
+    click.secho("Login credentials are valid only for 60 seconds. Please be quick.")
```

Comment:
nit: This is printed even when you have the `--connect` flag enabled. It's maybe a little confusing because you don't have to do anything in that case, but still get the message.

Reply:
Yes, but it is printed after the user closes the SSH connection. It's exactly like how the command works now.